src/components/dialogs/SaveErrorDialog.vue

```diff
@@ -66,27 +75,37 @@ export default class SaveErrorDialog extends Vue {
   // Getter definition for static type checking.
   readonly isRoleStaff!: boolean
 
-  // Prop containing filing name.
-  @Prop({ default: 'Filing' }) private filing: string
+  /** Prop containing filing name. */
+  @Prop({ default: 'Filing' }) private filingName: string
 
-  // Prop to display the dialog.
+  /** Prop to display the dialog. */
   @Prop() private dialog: boolean
 
-  // Prop to provide attachment selector.
+  /** Prop to provide attachment selector. */
   @Prop() private attach: string
 
-  // Prop to disable the Retry button.
+  /** Prop to disable the Retry button. */
   @Prop() private disableRetry: boolean
 
-  // Prop containing error messages.
+  /** Prop containing error messages. */
   @Prop({ default: () => [] }) private errors: object[]
 
-  // Prop containing warning messages.
+  /** Prop containing warning messages. */
   @Prop({ default: () => [] }) private warnings: object[]
 
   // Pass click events to parent.
   @Emit() private exit () { }
   @Emit() private retry () { }
   @Emit() private okay () { }
+
+  /** The number of errors in the passed-in array. */
+  private get numErrors (): number {
+    return this.errors?.length || 0
+  }
```

Comment:
This is the main change in this file: using getters to 1) simplify the template (HTML), and 2) increase performance by only calculating length once (since getters are cached).

Reply:
And 3) not crash if `warnings` or `errors` is ever null for any reason.